multinet/api/views/alttxt.py

```diff
@@ -0,0 +1,74 @@
+from typing import Type, Any, Union
+from pprint import pprint
+import json
+
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status, serializers
+from django.core.files.uploadedfile import UploadedFile
+
+from alttxt.enums import Level, Verbosity, Listable, Explanation, AggregateBy
+from alttxt.generator import AltTxtGen
+from alttxt.models import DataModel, GrammarModel
+from alttxt.parser import Parser
+from alttxt.tokenmap import TokenMap
+
+class AlttxtSerializer(serializers.Serializer):
+    verbosity = serializers.ChoiceField(choices=[e.value for e in Verbosity])
+    level = serializers.ChoiceField(choices=[e.value for e in Level])
+    explain = serializers.ChoiceField(choices=[e.value for e in Explanation])
```

Comment:
Since `Verbosity`, `Level`, and `Explanation` inherit from `Listable`, wouldn't it make sense to make these (for example) `choices=Verbosity.list()`?